custom/opm/filters.py

```diff
@@ -51,6 +70,56 @@ def get_hierarchy():
     return hierarchy
 
 
+def user_data_as_hierarchy():
```

Comment:
I have also compared the results of HSR with and without this PR, and it looks consistent